src/flb_scheduler.c

```diff
@@ -286,8 +286,13 @@ int flb_sched_request_create(struct flb_config *config, void *data, int tries)
     timer->data = request;
     timer->event.mask = MK_EVENT_EMPTY;
 
-    /* Get suggested wait_time for this request */
-    seconds = backoff_full_jitter((int)config->sched_base, (int)config->sched_cap, tries);
+    /* Get suggested wait_time for this request. If shutting down, set to 0. */
+    if (config->is_shutting_down == FLB_TRUE) {
```

Comment:
I think part of the coding style is that those variables that are used to hold boolean values using the FLB_TRUE and FLB_FALSE constants are used straight in if blocks without comparing them to the value (ie. `if (myvar)` or `if (!myvar)`) (I like explicit comparisons better but the other way is the standard and having an homogeneous codebase is important)